app/javascript/gobierto_dashboards/webapp/containers/costs/Home.vue

```diff
@@ -0,0 +1,138 @@
+        <p class="gobierto-dashboards-description">
+          {{ labelDescription3 }}
+        </p>
+      </div>
+      <Distribution
+        :data="groupData"
+        :year="yearFiltered"
+        :years="years"
+        @preventReload="injectRouter"
+      />
+      <Table
+        :items-filter="groupDataFilter"
+        :items="costDataFilter"
+        :year="yearFiltered"
+        :base-title="baseTitle"
+      />
+    </div>
+  </main>
+</template>
+<script>
+import Distribution from './Distribution.vue'
+import Table from './table/Table.vue'
+export default {
+  name: 'Home',
+  components: {
+    Distribution,
+    Table
+  },
+  data() {
+    return {
+      costData: this.$root.$data.costData,
+      groupData: this.$root.$data.groupData,
+      getSiteName: document.querySelector('[data-site-name]').getAttribute('data-site-name'),
+      labelTittle: I18n.t("gobierto_dashboards.dashboards.costs.title") || "",
+      labelDescription: '',
+      labelDescription2: I18n.t("gobierto_dashboards.dashboards.costs.description_2") || "",
+      labelDescription3: I18n.t("gobierto_dashboards.dashboards.costs.description_3") || "",
+      yearFiltered: "2018",
+      years: ['2018', '2019'],
+      costDataFilter: [],
+      groupDataFilter: []
+    }
+  },
+  watch: {
+    $route(to) {
+      if (to.name) {
+        this.injectRouter()
+      }
+    }
+  },
+  created() {
+      this.labelDescription = I18n.t("gobierto_dashboards.dashboards.costs.description", { entity_name: this.getSiteName }) || "";
+    const {
+      params: {
+        year: year
+      }
+    } = this.$route
+    let yearFiltered = year
+    if (!year) yearFiltered = '2018'
+    this.yearFiltered = yearFiltered
+
+    this.costDataFilter = this.costData.filter(element => element.year === yearFiltered)
+    this.groupDataFilter = this.groupData.filter(element => element.year === yearFiltered)
+
+    this.costDataFilter = this.costDataFilter.sort((a, b) => (a.cost_total > b.cost_total) ? -1 : 1)
+    this.groupDataFilter = this.groupDataFilter.sort((a, b) => (a.cost_total > b.cost_total) ? -1 : 1)
+
+    this.baseTitle = document.title;
+  },
+  mounted() {
+    this.injectRouter()
+  },
+  methods: {
+    onChangeFilterYear(value) {
+      this.injectRouter()
+      let year
+      if (value === '2018') {
+        year = value
+      } else {
+        year = value.target.value
+        this.yearFiltered = value.target.value
+      }
+      this.costDataFilter = this.costData.filter(element => element.year === year)
+      this.groupDataFilter = this.groupData.filter(element => element.year === year)
+      // eslint-disable-next-line no-unused-vars
+      this.$router.push(`/dashboards/costes/${year}`).catch(err => {})
+    },
+    injectRouter() {
+      const bubbleLinks = document.querySelectorAll('.bubbles-links')
+      bubbleLinks.forEach(bubble => bubble.addEventListener('click', (e) => {
+        const {
+          target: {
+            __data__: {
```

Comment:
Are you getting directly to the `d3` object?

Reply:
Yep, d3 bubbles contains a link to section with the order and year.